Gemfile

```diff
@@ -13,6 +13,6 @@ gem 'zeus', require: false
 
 # YARD
 gem 'fssm'
-gem 'pygments.rb'
+gem 'rouge'
```

Comment:
I'm not sure why we do not specify versions in the Gemfile, so continuing with it.
But i think it would be good to specify for all.

Reply:
The versions are locked in `Gemfile.lock` and `gemfiles/*.gemfile.lock`, and all of those are committed to the repo, so that's what keeps things in check from a development perspective. It would only be a potential problem if someone were to run `bundle update` and update everything across the board, but that happens pretty rarely.

Reply:
Yes, thats true.
Just mentioning few points for thoughts.

- But using `pessimistic locking (~>)` is helpful in many ways like when bundle update is executed in case we need some fix by a gem and we have defined the version to minor `Major.minor.patch`, it will update the gem but do not introduce breaking changes.
- Defining also helps in knowing if a gem has some incompatible version with our use case, if any.
- In case `Gemfile.lock` has conflicts then one usually removes the content and regenerate the lock file, this might also introduce unwarranted versions as lock content was removed.

_That said, we can continue with this approach as it all depends on how the gem is being maintained._